query-parser/src/lib.rs

```diff
@@ -8,6 +8,11 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+#![allow(unused_imports)]
+
+#[macro_use(assert_parses_to)]
+extern crate parser_utils;
```

Comment:
This should be inside the testing module that I introduced in f1a55c9. That should fix the unused import warning that you've allowed above.